pkgs/development/python-modules/keystoneauth-websso/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  keystoneauth1,
+  multipart,
+  pythonOlder,
+  setuptools,
+  poetry-core,
+}:
+
+buildPythonPackage rec {
+  pname = "keystoneauth_websso";
+  version = "0.2.0";
+
+  disabled = pythonOlder "3.8";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-84GtBYOh/kYmMMk5Voyflw3NrBuxkV/Vj+ih8SbwzAo=";
+  };
```

Comment:
fetchFromGitHub is preferred unless there is a compelling reason to use Pypi

Reply:
changed, but not sure if I'm using the `version` and `tag` correctly - please let me know if that needs to change (it's my first package)

Reply:
Yup. Looks good